website/docs/guides/explorer-qs.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Quickstart for the dbt Explorer workshop"
+description: "Use this guide to build and define metrics, set up the dbt Cloud Semantic Layer, and query them using Google Sheets."
+sidebar_label: "Quickstart dbt Explorer"
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Explorer', 'Snowflake', 'dbt Cloud','Quickstart']
+keywords: ['dbt Explorer','Mesh','dbt Cloud', 'Snowflake', 'Multi-Project']
+level: 'Beginner'
+recently_updated: true
+---
+
+
+## Introduction
+Unlock the power of [dbt Explorer](/docs/collaborate/explore-projects) in this hands-on workshop designed for analytics engineers, data analysts, stakeholders, and data leaders! Dive into a production-level dbt Mesh implementation and discover how to explore your data workflows.⁠ Whether you're looking to streamline your data operations, improve data quality, or self-serve information about your data platform, this workshop will equip you with the tools and knowledge to take your dbt projects to the next level.
+
+**You’ll learn how to:**
+- Navigate multiple dbt projects using dbt Explorer
+- Self-serve on data documentation
+- Trace dependencies at the model and column level
+- Identify opportunities to improve performance and data quality
+
+**Prerequisites:**
+- Familiarity with data platforms
+
+## Setup
+Now we’ll be creating your dbt Cloud account and connecting it to a data warehouse. 
+- Go to this URL:  https://cloud.getdbt.com/coalesce-workshop-signup
+  - If you are already signed into dbt Cloud, sign out first!
+  - Enter your first name and last name
+  - Choose the “Exploring a dbt Mesh implementation with dbt Explorer” option.
+- Agree to the terms of service and click the “Complete Registration” button. Wait about 30 seconds, and you’ll be in the dbt Cloud project for this course, already connected to a data warehouse!
```

Comment:
there's also a password option - i think we should mention they can enter the password given to them at the workshop (i assume)?

Reply:
@MichelleBaird-DBT this is what i saw:

![Screenshot 2024-09-26 at 11 22 22](https://github.com/user-attachments/assets/374a35fe-38d4-4bbc-8d53-649e328ca817)